LICENSE

```diff
@@ -178,15 +179,15 @@ Apache License
    APPENDIX: How to apply the Apache License to your work.
 
       To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
+      boilerplate notice, with the fields enclosed by brackets "[]"
       replaced with your own identifying information. (Don't include
       the brackets!)  The text should be enclosed in the appropriate
       comment syntax for the file format. We also recommend that a
       file or class name and description of purpose be included on the
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-    Copyright 2018 HM Revenue & Customs
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
This line _should_ be changed. As per the first para...

> ...attach the following boilerplate notice, **with the fields enclosed by brackets "[]" replaced with your own identifying information**. (Don't include the brackets!)...

Reply:
ah hang on, i see what you mean. as long as the source files have that notice (which as you said, they do) then we're all good.